src/core.cc

```diff
@@ -168,4 +169,22 @@ Napi::Value Core::LoadNetwork(const Napi::CallbackInfo& info) {
   return deferred.Promise();
 }
 
+Napi::Value Core::GetAvailableDevices(const Napi::CallbackInfo& info) {
+  Napi::Env env = info.Env();
```

Comment:
Could you please check whether the input is valid?

Reply:
Done.